cmd/main.go

```diff
@@ -118,18 +116,18 @@ func setupK8SClient(kubeConfigFile *string, leaderElect *bool) kubernetes.Interf
 		}
 		k8sClient, err := k8s.NewOutOfClusterClient(*kubeConfigFile)
```

Comment:
This could be further simplified to just `return k8s.NewOutOfClusterClient(*kubeConfigFile)`